antarest/study/web/raw_studies_blueprint.py

```diff
@@ -96,11 +96,16 @@ def get_study(
             extra={"user": current_user.id},
         )
         parameters = RequestParameters(user=current_user)
-        output = study_service.get(uuid, path, depth, formatted, parameters)
+
+        resource_path = pathlib.PurePosixPath(path)
+        output = study_service.get(uuid, str(resource_path), depth=depth, formatted=formatted, params=parameters)
 
         if isinstance(output, bytes):
-            resource_path = pathlib.Path(path)
-            suffix = resource_path.suffix.lower()
+            # Guess the suffix form the target data
+            parent_cfg = study_service.get(uuid, str(resource_path.parent), depth=2, formatted=True, params=parameters)
```

Comment:
I do not understand why you use depth=2, can you explain it to me by responding to this comment ?

Reply:
I only need to get the properties' value.